src/runtime_src/core/tools/xbutil2/SubCmdValidate.cpp

```diff
@@ -1349,9 +1349,21 @@ get_platform_info(const std::shared_ptr<xrt_core::device>& device,
 {
   auto bdf = xrt_core::device_query<xrt_core::query::pcie_bdf>(device);
   ptTree.put("device_id", xrt_core::query::pcie_bdf::to_string(bdf));
-  ptTree.put("platform", xrt_core::device_query<xrt_core::query::rom_vbnv>(device));
-  ptTree.put("sc_version", xrt_core::device_query<xrt_core::query::xmc_sc_version>(device));
-  ptTree.put("platform_id", (boost::format("0x%x") % xrt_core::device_query<xrt_core::query::rom_time_since_epoch>(device)));
+
+  boost::property_tree::ptree platform_report;
+  boost::property_tree::ptree empty_ptree;
```

Comment:
It would be great if this value was a "const".  My guess is that line 1358 will fail if it is....don't 100% know.

Reply:
I can't declare `platform_report` as const :(